experiments/ClimaEarth/run_moist_held_suarez.jl

```diff
@@ -0,0 +1,354 @@
+# # Moist Held-Suarez
+
+redirect_stderr(IOContext(stderr, :stacktrace_types_limited => Ref(false)))
+
+#=
+## Configuration initialization
+=#
+
+#=
+### Package Import
+=#
+
+## standard packages
+import Dates
+import YAML
+
+# ## ClimaESM packages
+import ClimaAtmos as CA
+import ClimaComms
+import ClimaCore as CC
+
+# ## Coupler specific imports
+import ClimaCoupler
+import ClimaCoupler:
+    BCReader,
+    ConservationChecker,
+    Checkpointer,
+    Diagnostics,
+    FieldExchanger,
+    FluxCalculator,
+    Interfacer,
+    Regridder,
+    TimeManager,
+    Utilities
+
+pkg_dir = pkgdir(ClimaCoupler)
+
+#=
+### Helper Functions
+=#
+
+## helpers for component models
+include("components/atmosphere/climaatmos.jl")
+
+## helpers for user-specified IO
+include("user_io/user_diagnostics.jl")
+include("user_io/user_logging.jl")
+
+include("user_io/io_helpers.jl")
+
+#=
+### Setup simulation parameters
+Here we follow Thatcher and Jablonowski (2016).
+=#
+
+## run names
+run_name = "moist_held_suarez"
+coupler_output_dir = "$run_name"
+const FT = Float64
+restart_dir = "unspecified"
+restart_t = Int(0)
+
+## coupler simulation specific configuration
+Δt_cpl = Float64(400)
+t_end = "1000days"
+tspan = (Float64(0.0), Float64(time_to_seconds(t_end)))
+start_date = "19790301"
+hourly_checkpoint = true
+
+## namelist
+config_dict = Dict(
+    # general
+    "FLOAT_TYPE" => string(FT),
+    # file paths
+    "atmos_config_file" => nothing,
+    "coupler_toml_file" => nothing,
+    "coupler_output_dir" => coupler_output_dir,
+    "mode_name" => "",
+    "run_name" => run_name,
+    "atmos_config_repo" => "ClimaAtmos",
+    # timestepping
+    "dt" => "$(Δt_cpl)secs",
+    "dt_save_to_sol" => "1days",
+    "t_end" => t_end,
+    "start_date" => "19790301",
+    # domain
+    "h_elem" => 4,
+    "z_elem" => 10,
+    "z_max" => 30000.0, # semi-high top
+    "dz_bottom" => 300.0,
+    "nh_poly" => 4,
+    # output
+    "dt_save_to_sol" => "1days",
+    # numerics
+    "apply_limiter" => false,
+    "viscous_sponge" => false,
+    "rayleigh_sponge" => false,
+    "vert_diff" => "true",
+    "hyperdiff" => "ClimaHyperdiffusion",
+    # run
+    "job_id" => run_name,
+    "surface_setup" => "PrescribedSurface",
+    # diagnostic (nested with period and short_name)
+    "output_default_diagnostics" => false,
+    "diagnostics" => [
+        Dict(
+            "short_name" =>
+                ["mse", "lr", "mass_strf", "stab", "vt", "egr", "ua", "va", "wa", "ta", "rhoa", "pfull"],
+            "period" => "6hours",
+            "reduction" => "inst",
+        ),
+    ],
+    # held-suarez specific
+    "forcing" => "held_suarez", # Newtonian cooling already modified for moisture internally in Atmos
+    "precip_model" => "0M",
+    "moist" => "equil",
+    "prognostic_surface" => "PrescribedSurfaceTemperature",
+    "turb_flux_partition" => "CombinedStateFluxesMOST",
+)
+# TODO: may need to switch to Bulk fluxes
```

Comment:
why would we need to switch?

Reply:
Though less accurate, the bulk fluxes are what's used in the original paper. We use MOST, so we might want to switch to the bulk fluxes if we want to follow the exact setup as in the paper. I want to leave this decision for the refinement stage of this project though.

Reply:
sounds good, thank you for clarifying!